src/views/evaluation/evaluations.js

```diff
@@ -0,0 +1,113 @@
+function EvaluationsPage(options = {}) {
+    const headings = { 
+        headings: ["Nume", "Technology", "Nivel", ""] 
+        }
+
+    const rows =[
+        {
+            name: "Adrian Popescu",
+            technology: "Javascript",
+            level: "mid"
+        },
+    
+        {
+            name: "Adrian Popescu2",
+            technology: "Javascript2",
+            level: "mid2"
+        },
+    
+        {
+            name: "Adrian Popescu3",
+            technology: "Javascript3",
+            level: "mid3"
+        },
+        
+        {
+            name: "Adrian Popescu4",
+            technology: "Javascript4",
+            level: "mid4"
+        }
+    ];
+
+    return `
+    ${nav()}
+    ${EvaluationsTable({
+        items: rows,
+        itemHeadings: headings
+    })}
+    ${Footer()}
+   `
+}
+
+// function EvaluationsTableHeader(options, tag) {
+//     return `<${tag}>${options.join(`<${tag}></${tag}>`)}<${tag}>`;
+// }
+
+function EvaluationsTableHeader(options) {
+    const headings = [];
+    options.headings.forEach(function(el) {
+        const generatedEl = `<th>${el}</th>`;
+        headings.push(generatedEl);
+    });
+    const headingsEl = `${headings.join('')} `;
+    return `
+    <tr>
+        ${headingsEl}
+    </tr>
+    `;
+}
+
+
+function EvaluationTableRow(options={}){
+   return `
+<tr>
+    <td>${options.name}</td>
+    <td>${options.technology}</td>
+    <td>${options.level} 2</td>
+    <td>Detalii <button></button></td>
+</tr>
+    `
+}
+
+function EvaluationTableBody(options = {}) {
+
+const rowsElements = [];
+
+options.items.forEach(function(rowObj){
```

Comment:
Couldn't agree more, .map is magical but I'm just getting my head around it. So please sir have mercy if the first draft of the code is not getting straight to .map